fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java

```diff
@@ -93,6 +93,9 @@ public class PropertyAnalyzer {
 
     public static final String PROPERTIES_INMEMORY = "in_memory";
 
+    public static final String PROPERTIES_AUTO_BUCKET = "auto_bucket";
```

Comment:
I didn't see this property in the example you gave.

Reply:
the properties is not from create table stmt syntax. It generated in rewrite